src/test/resources/conf/parallel.conf.json

```diff
@@ -0,0 +1,35 @@
+{
+  "server": "hub-cloud.browserstack.com",
+  "user": "BROWSERSTACK_USERNAME",
+  "key": "BROWSERSTACK_ACCESS_KEY",
+  "capabilities": {
+    "bstack:options": {
+      "sessionName": "BStack parallel serenity",
+      "buildName": "browserstack-build-1",
+      "projectName": "Test App",
+      "debug": true
+    }
+  },
+  "environments": {
+    "chrome": {
+      "browserName": "chrome"
+    },
+    "firefox": {
+      "browserName": "firefox"
+    },
+    "safari": {
+      "browserName": "safari",
+      "bstack:options": {
+        "os": "OS X",
+        "osVersion": "High Sierra"
+      }
+    },
+    "ie": {
```

Comment:
Pls confirm is this IE version works with the sample demo site (in some cases demo site breaks on IE)

Reply:
We were not using IE browser, so removed it..